online-shop/src/app/CartService.ts

```diff
@@ -0,0 +1,53 @@
+import { Observable, of } from "rxjs";
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from "@angular/core";
+import { IProduct } from "./IProduct";
+import { IProductWrapper } from "./IProductWrapper";
+
+
+@Injectable({
+    providedIn: 'root'
+})
+
+export class CartService {
```

Comment:
**suggestion:** Try to respect the angular styling guide: https://angular.io/guide/styleguide. Thus the file itself (not the class name) should be named as `cart.service` not `CartService`